### ISSUE TITLE
Esp8266 not sending data

### ISSUE THREAD

**Issue by richyxi** [not-matt/Systematic-LEDs#73]
The ESP8266 gets connected to the internet but it's no sending data to the led strip even when is connected to the RX pin, what could it be?

**Comment by Andriejus** [comment]
Maybe LED strip is damaged? If I were you I would upload simple sketch to esp8266 for controlling the ws2812 led strip and check if its working.

**Comment by richyxi** [comment]
Yeah I already did that, it works with other programs, how's your led strip
connected? Your are using RX?

**Comment by Andriejus** [comment]
yes RX. If I remember correct it was pin number 7 in Arduino.ino sketch.  #define DATA_PIN      7
I'm using MAC+esp8266 and you are using raspberry yes? I have no experience with raspberry..

**Comment by richyxi** [comment]
Are u sharing a gnd connection between led and esp?

**Comment by Andriejus** [comment]
Yes. Ground conmection is very important.

**Comment by richyxi** [comment]
Mm okay, your powering you esp with a USB?

**Comment by Andriejus** [comment]
I have two esp and two led strips.  one esp is powered through usb and another with power supply through “vin” pin.
